flytekit/annotated/task.py

```diff
@@ -472,7 +593,7 @@ def execute(self, **kwargs) -> Any:
 
 class DynamicWorkflowTask(PythonFunctionTask):
     def __init__(self, dynamic_workflow_function: Callable, metadata: _task_model.TaskMetadata, *args, **kwargs):
-        super().__init__(dynamic_workflow_function, metadata, *args, **kwargs)
+        super().__init__(dynamic_workflow_function, metadata, task_type="dynamic_task", *args, **kwargs)
```

Comment:
Can we copy in the current strings in `SdkTaskType` so that the rest of the system respects it as well?

Reply:
sure will do